package.json

```diff
@@ -1,12 +1,13 @@
 {
   "name": "evaporate",
-  "version": "2.1.4",
+  "version": "3.0.0-rc",
   "description": "Javascript library for browser to S3 multipart resumable uploads for browsers and with Node FileSystem (fs) Stream Support",
-  "main": "evaporate.js",
+  "main": "codemod/dist/build.js",
```

Comment:
You might want to use [rollup](https://github.com/rollup/rollup) to provide `cjs` and `es` bundle.

Reply:
I would add additional tooling in a different PR.